apps/passport-server/src/apis/pretix/genericPretixAPI.ts

```diff
@@ -579,9 +595,11 @@ const GenericPretixEventSettingsSchema = z.object({
   attendee_emails_required: z.boolean()
 });
 
-const GenericPretixCategorySchema = z.object({
+// @rob - what are product categories?
+const GenericPretixProductCategorySchema = z.object({
   id: z.number(),
   is_addon: z.boolean()
+  // @rob - should we load category names?
```

Comment:
Doesn't look like it would hurt, but none of the category data is persisted anywhere. It's only used for validation.